### DISCUSSION TITLE
Disable api bodyParser for one specific route in [[...params]].ts?

### DISCUSSION THREAD

**Discussion by alaughlin** [instantcommerce/next-api-decorators#577] [Q&A]
So I have a general [[...params]].ts file to handle webhooks from a few different services, and one of the routes defined inside it is to handle a webhook call from Stripe, which requires me to disable nextjs's api bodyparser.

I know I can add this above my handler class:
```
export const config = {
    api: {
        bodyParser: false,
    },
};
```

However that applies it all the routes defined in the handler. Is there any way to apply this config to just one route?

**Comment by ggurkal** [comment]
Hi @alaughlin 

Unfortunately that's not possible as it's handled by Next.js.
